<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RazorClockConfiguration</class>
 <widget class="QDialog" name="RazorClockConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Razor Clock Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="timeGB">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="showSecondsCB">
        <property name="text">
         <string>&amp;Show seconds</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="ampmClockCB">
        <property name="text">
         <string>12 &amp;hour style</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="timeFontL">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Font</string>
        </property>
        <property name="buddy">
         <cstring>timeFontB</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="timeFontB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dateGB">
     <property name="title">
      <string>Date</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="showDateCB">
        <property name="text">
         <string>Show &amp;date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="dateFormatL">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>D&amp;ate format</string>
        </property>
        <property name="buddy">
         <cstring>dateFormatCOB</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="dateFormatCOB">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="dateFontB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dateFontL">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Fon&amp;t</string>
        </property>
        <property name="buddy">
         <cstring>dateFontB</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="dateOnNewLineCB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show date in &amp;new line</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useThemeFontCB">
     <property name="text">
      <string>&amp;Use theme fonts</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="calendarGB">
     <property name="title">
      <string>Calendar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
      <item row="0" column="0">
       <widget class="QLabel" name="firstDayOfWeekL">
        <property name="text">
         <string>First day of &amp;week</string>
        </property>
        <property name="buddy">
         <cstring>firstDayOfWeekCB</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="firstDayOfWeekCB">
        <property name="currentIndex">
         <number>6</number>
        </property>
        <property name="maxVisibleItems">
         <number>7</number>
        </property>
        <item>
         <property name="text">
          <string>Monday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tuesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wednesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thursday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Friday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saturday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sunday</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>RazorClockConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>RazorClockConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useThemeFontCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeFontB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDateCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateOnNewLineCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDateCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateFormatCOB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDateCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateFormatL</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useThemeFontCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeFontL</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
